### ISSUE TITLE
Kitchen Sink : Complete Books_Store [user , admin] Website 

### ISSUE THREAD

**Issue by Bharath-KumarReddy** [aidenybai/million#713]
### Describe the feature

complete user , Admin ------ functionalities 

Sign up , Login using JWT Authentication;   Forgot password_Uisng node mailer

after user logged in then his section the books which was posted by the admin will be present he can download and read it ;

Serach Functionality-----
User can search Any books based on his interest ----for this I am using  Google API;

and User can request a book to the admin , in case if he want any books ;

from the ADMIN-----
he can see all the users , he can post the books , delete the books , Update the books and he can also view all the requests made by all the users with his names and he can resolve those ;

and this is build with good Ui , I am using react-toastify library and Bootstrap, CSS , Nodejs , Mongodb , express , React.js

can you please consider this and assign it , I am very much interested to participate myself in hacktoberfest_23 : )

### Additional information

- [X] Would you be willing to help implement this feature?

**Comment by tobySolutions** [comment]
Amazing! Go ahead @Bharath-KumarReddy

**Comment by tobySolutions** [comment]
Any updates on this @Bharath-KumarReddy?

**Comment by Bharath-KumarReddy** [comment]
yes , I had almost done, but I am getting a problem that if I made a PR in the kitchen sink then vercel deployment is failing :)

**Comment by Timonwa** [comment]
> yes , I had almost done, but I am getting a problem that if I made a PR in the kitchen sink then vercel deployment is failing :)

Hi @Bharath-KumarReddy, if it fails, it is because the app failed to build. First of all, could you update your repo and run the cleanup script? Then, could you run the build script on your local repo? If it passes, then you no longer have an issue. If it doesn't work, then please check your terminal for errors and fix them. Then try again. If it passes locally, then it should pass on your PR.

if, after doing all these you still have an issue, let us know.

**Comment by tobySolutions** [comment]
Any updates on this @Bharath-KumarReddy? How's it going?

**Comment by Bharath-KumarReddy** [comment]
Hey , I had made a PR #734 once please check it : )

**Comment by Bharath-KumarReddy** [comment]
User side --------



https://github.com/aidenybai/million/assets/127650446/af3e08c8-a723-4925-8c38-d25e810c240b


Admin side -------


https://github.com/aidenybai/million/assets/127650446/8c3717f3-319a-4d55-942c-ebd3adbabb77

**Comment by tobySolutions** [comment]
> Hey , I resolved this and made a PR #734 once please check it : )


Awesome, gonna check this out soon.

**Comment by tobySolutions** [comment]
I dropped a couple of comments on your PR @Bharath-KumarReddy, any updates regarding those, yet?
